Mu2eUtilities/inc/MueXSpectrum.hh

```diff
@@ -0,0 +1,58 @@
+#ifndef Mu2eUtilities_MueXSpectrum_hh
+#define Mu2eUtilities_MueXSpectrum_hh
+
+// CLHEP includes
+#include "CLHEP/Vector/LorentzVector.h"
+
+// C++ includes
+#include <utility>
+
+namespace mu2e {
+
```

Comment:
As declared, I believe the function f is visible within the entire Mu2e scope when this file is included, and it would be easy to come up with a name clash at some point. If using an anonymous namespace isn't practical (should really try to make this work), then I would change the name to something much less easy to re-declare by accident (e.g. f_mu2ex_gen or anything not straightforward).